docs/overview.md

```diff
@@ -7,117 +7,101 @@ sidebar_label: Overview
+ 
+
+### What are some key differentiators of OpenEBS?
 
 Some key aspects that make OpenEBS different compared to other traditional storage solutions:
 - Built using the _micro-services architecture_ like the applications it serves. OpenEBS is itself deployed as a set of containers on Kubernetes worker nodes. Uses Kubernetes itself to orchestrate and manage OpenEBS components.
 - Built completely in userspace making it highly portable to run across _any OS/platform_.
 - Completely intent-driven, inheriting the same principles that drive the _ease of use_ with Kubernetes.
-- OpenEBS supports a range of storage engines so that developers can deploy the storage technology appropriate to their application design objectives. Distributed applications like Cassandra can use the _LocalPV_ engine for lowest latency writes. Monolithic applications like MySQL and PostgreSQL can use _Mayastor_ or _cStor based on ZFS_ for resilience. Streaming applications like Kafka can use the NVMe engine [Mayastor](https://github.com/openebs/Mayastor) for best performance in edge environments. Across engine types, OpenEBS provides a consistent framework for high availability, snapshots, clones and manageability.
+- OpenEBS supports a range of storage engines so that developers can deploy the storage technology appropriate to their application design objectives. Distributed applications like Cassandra can use the _LocalPV_ engine for lowest latency writes. Monolithic applications like MySQL and PostgreSQL can use _Mayastor built using NVMe and SPDK_ or _cStor based on ZFS_ for resilience. Streaming applications like Kafka can use the NVMe engine [Mayastor](https://github.com/openebs/Mayastor) for best performance in edge environments. 
 
-An added advantage of being a completely Kubernetes native solution is that administrators and developers can interact and manage OpenEBS using all the wonderful tooling that is available for Kubernetes like kubectl, Helm, Prometheus, Grafana, Weave Scope, etc.
+Also looking at the <a href="https://github.com/openebs/openebs/blob/master/ADOPTERS.md" target="_blank">OpenEBS Adoption stories</a>, the top reasons driving users towards OpenEBS are:
+- Being Portable across all Kubernetes distributions - On-premise and Cloud.
+- Increased Developer and Platform SRE Productivity.
+- Easy to use compared to other solutions.
+- Excellent Community Support.
+- Open Source and Free.
 
-Check out what users of OpenEBS have to say about their experience in <a href="https://github.com/openebs/openebs/blob/master/ADOPTERS.md" target="_blank">OpenEBS Adoption stories</a>.
 
-## Types of OpenEBS Storage Engines
+## Local Volumes 
```

Comment:
Is it good to mention its volume binding mode as `WaitForFirstConsumer`? This parameter says all for a Kubernetes user.

Reply:
we support both. Will take this up after some reviews through slack.